adapters/vrtcal/vrtcal.go

```diff
@@ -101,10 +101,17 @@ func getReturnTypeForImp(impID string, imps []openrtb2.Imp) (openrtb_ext.BidType
 				return openrtb_ext.BidTypeBanner, nil
 			}
 
+
 			if imp.Video != nil {
 				return openrtb_ext.BidTypeVideo, nil
 			}
 
+
```

Comment:
nit - there is already a blank line above line 109. Please remove this blank line.